docs/configuring.rst

```diff
@@ -8,6 +8,93 @@ contact the Globus Compute Team via Slack.
    All configuration examples below must be customized for the user's
    allocation, Python environment, file system, etc.
 
+GlobusComputeEngine
+^^^^^^^^^^^^^^^^^^^
+
+``GlobusComputeEngine`` is the execution backend that ``GlobusCompute`` uses
```

Comment:
Is there any value in linking to the GCE API page?  Much the same way we do in the (Compute Executor](https://globus-compute.readthedocs.io/en/stable/executor.html) documentation?

Reply:
Yes, I've updated the entire doc to link to both GCE and HTEX references, esp since we borrow heavily from it.